### PR TITLE
Media Library: Hide edit/delete options if lacking permissions

### LINKED ISSUE #8221: Media Library: Hide edit/delete options if lacking permissions
## Feature Description

Originally reported in https://github.com/google/web-stories-wp/issues/7755#issuecomment-849804718

Users with the "Contributor" role are shown options to edit and delete items in the media library on the left hand side, but because they don't actually have permissions to do that it doesn't work of course.

We should hide the "..." menu if the user does not have permission to edit media.

I suppose checking for `hasUploadMediaAction` is good enough for now, but ideally this is better handled by #8276 in the future.

## Additional Context

Current behavior:

https://user-images.githubusercontent.com/841956/119869314-5fad8c80-bf20-11eb-933f-4f7e2a0ddb29.mov

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Users with the "Contributor" role are shown options to edit and delete items in the media library on the left hand side, but because they don't actually have permissions to do that it doesn't work of course.

We should hide the "..." menu if the user does not have permission to edit media.

### QA

- [ ] This is a non-user-facing change and requires no QA

This PR can be tested by following these steps:

1. Login as contributor. 
2. Create a new story. 
3. Hover over a media library element like image / video. 
4. Do not see `...` menu (aka it's not possible to edit the media) 

### UAT

- [x] UAT should use the same steps as above.

This PR can be tested by following these steps:

1.

## Checklist

- [x] This PR addresses an existing issue and I have linked this PR to it in ZenHub
- [x] I have tested this code to the best of my abilities
- [x] I have verified accessibility to the best of my abilities ([docs](https://github.com/google/web-stories-wp/blob/main/docs/accessibility-testing.md))
- [x] I have verified i18n and l10n (translation, right-to-left layout) to the best of my abilities
- [x] This PR contains automated tests (unit, integration, and/or e2e) to verify the code works as intended ([docs](https://github.com/google/web-stories-wp/tree/main/docs#testing))
- [x] I have added documentation where necessary
- [x] I have added a matching `Type: XYZ` label to the PR

---

Fixes #8221

### CODE REVIEW

File: assets/src/edit-story/components/library/panes/media/local/mediaPane.js [hunk 1]
```diff
@@ -360,6 +361,7 @@ function MediaPane(props) {
         ) : (
           <PaginatedMediaGallery
             providerType="local"
+            hasUploadMediaAction={hasUploadMediaAction}
```
Comment (swissspidy) [inline assets/src/edit-story/components/library/panes/media/local/mediaPane.js]: Thoughts on renaming this prop everywhere to something more concise?

```suggestion
            canEditMedia={hasUploadMediaAction}
```

Comment (spacedmonkey) [reply assets/src/edit-story/components/library/panes/media/local/mediaPane.js]: Yeah, works for me.

**Comment by swissspidy** [comment]
The `should render dropdown menu's more icon for uploaded video` unit test now needs updating.

I suggest creating two tests, one with the capability, one without.

**Review by swissspidy — APPROVED** [review]
Nice!

**Pull request merged**